php-checks/src/main/java/org/sonar/php/checks/utils/type/StaticFunctionCall.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * SonarQube PHP Plugin
+ * Copyright (C) 2010-2018 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package org.sonar.php.checks.utils.type;
+
+import org.sonar.php.checks.utils.namespace.QualifiedName;
+
+import static org.sonar.php.checks.utils.namespace.QualifiedName.qualifiedName;
+
+public class StaticFunctionCall {
+  private QualifiedName callee;
+  private String functionName;
+
+  private StaticFunctionCall(QualifiedName callee, String functionName) {
+    this.callee = callee;
+    this.functionName = functionName;
+  }
+
+  public boolean matches(QualifiedName className, String memberName) {
+    return functionName.equals(memberName) && callee.equals(className);
```

Comment:
We should keep in mind that we will have to use `callee.equalsIgnoreCase(className)` and also fix QualifiedName#equals